src/v2/spec.yaml

```diff
@@ -3158,6 +3158,26 @@ paths:
                 $ref: "#/components/schemas/UsersResponse"
         "400":
           $ref: "#/components/responses/400Response"
+  /farcaster/user/power-lite:
+    get:
+      tags:
+        - User
+      summary: Fetch power user FIDs
+      description: Fetches power users and respond in a backwards compatible format to Warpcast's deprecated power badge endpoint.
+      externalDocs:
+        url: https://docs.neynar.com/reference/user-power-lite
+      operationId: power-users-lite
+      responses:
+        "200":
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UsersResponse"
```

Comment:
`/farcaster/user/power-lite` API returns a list of fids inside the result object whereas UsersResponse returns a list of Neynar's User object with the cursor.

Afaik there isn't a schema for this response (Bt please check), so you might need to create a new schema and link it here.

I would suggest creating 2 schemas `Fids` and `PowerUserLiteResponse`

Reply:
Good catch! Since this API is meant to be a one-off response structure (we're just doing it to be backwards compatible with the deprecated warpcast power users api), I'm going to inline the schema. lmk if you feel strongly about creating the 2 schemas instead though